iphone/metadata/en-US/release_notes.txt

```diff
@@ -1,27 +1,34 @@
-We and our contributors have prepared several Christmas/New Year presents for you:
+We have prepared several Christmas/New Year presents for you:
+
 • Outdoors map style for hiking, cycling, and exploring the Nature
-• Search for addr:place village addresses without streets in Austria, Czechia, Germany, Poland, Slovakia
+• Search for village addresses without streets in Austria, Czechia, Germany, Poland, Slovakia and some other countries
 • Type "skiing" or "ski run" in the search to find downhill and Nordic pistes
 • OpenStreetMap map data as of December 13
 • Added a button to see photos, reviews, and prices for some hotels. Every booking made through the app contributes to the development of the project!
 
 Other notable fixes and improvements:
 • Fixed several crashes and freezing on iOS 12, 13, 14
 • Multiple KML files can be imported from a single KMZ file
-• A credit card is displayed for places with ATM
-• A dollar emoji is displayed where a fee is requred
+• Less "gaps" in highways on the World map
+• Removed very short ferry lines from the World map
+• A credit card is displayed for selected places with ATM
+• A dollar emoji is displayed where a fee is required
```

Comment:
```suggestion
• A credit card is displayed in Place Page for POIs with ATM
• A dollar emoji is displayed in Place Page if a fee is required
```

Reply:
Users are not familiar with the Place Page concept. We use it internally. I've added "selected places" above, to keep the context.